main.go

```diff
@@ -118,11 +162,43 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Set up consistent hashing to shard reports over all of our exporters.
+	// This is arbitrarily based on the assumption that 97 exporters is a reasonable maximum for now.
+	// This could be made configurable in the future if actual usage requires it.
+	consistentCfg := consistent.Config{
+		PartitionCount:    97,
```

Comment:
So with exactly 97 exporters, we would end up with a 1 to 1 matching or partition to exporter or am I getting this wrong?

Reply:
Right. The partition count is recommended to be prime, but is otherwise arbitrary, so rather than making it configurable immediately I figured 97 was probably enough to start